assets/css/components/cardInstructor.css

```diff
@@ -106,21 +108,21 @@
 .item .card .content .knowledge ul li {
   list-style: none;
   padding: 5px 10px;
-  background-color: var(--color);
+  background-color: #222;
```

Comment:
Acredito que, se a cor for neutra fica melhor para visualização, menos colorido/poluído.

Reply:
verdade